recipes-devtools/python/python3-pycuda_2020.1.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Python wrapper for Nvidia CUDA"
+HOMEPAGE = "http://mathema.tician.de/software/pycuda"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5f3f85642a528c8fca0fe71e7a031203"
+
+SRC_URI = "https://files.pythonhosted.org/packages/46/61/47d3235a4c13eec5a5f03594ddb268f4858734e02980afbcd806e6242fa5/pycuda-${PV}.tar.gz"
+
+SRC_URI[sha256sum] = "effa3b99b55af67f3afba9b0d1b64b4a0add4dd6a33bdd6786df1aa4cc8761a5"
+
+S = "${WORKDIR}/pycuda-${PV}"
+
+inherit  cuda distutils3
```

Comment:
Could you please use Yocto class `pypi` and remove `SRC_URI` ?
You could add `SRC_URI[md5sum]` as well

Reply:
Maybe you need to define `PYPI_PACKAGE` here, I'm not sure.

Reply:
There is an additional space between inherit and cuda

Reply:
Dropping this one anyway.